engine/src/main/java/org/terasology/engine/paths/PathManager.java

```diff
@@ -188,6 +196,24 @@ public void useDefaultHomePath() throws IOException {
         updateDirs();
     }
 
+    /**
+     * Gives user the option to manually choose home path.
+     * @throws IOException Thrown when required directories cannot be accessed.
+     */
+    public void chooseHomePathManually() throws IOException {
```

Comment:
Should/can this be part of `useDefaultHomePath`?

Reply:
It could be but then it would have to catch the exception inside the `useDefaultHomePath`, moreover this provides a much modular approach and a generic solution to all further weird OSeS